cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/2_datasource_table.spec.js

```diff
@@ -294,20 +291,16 @@ if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
       });
     });
     describe('Selection & Deletion', () => {
-      it('should not select any rows by default & delete button should be disabled', () => {
+      it('should not select any rows by default & delete button should not exist', () => {
         cy.getElementByTestId('deleteDataSourceConnections').should(
-          'be.disabled'
-        );
-        cy.getElementByTestId('deleteDataSourceConnections').should(
-          'contain.text',
-          'Delete  '
+          'not.exist'
```

Comment:
The current render behavior is: when there is no selection, it will not render the deletion button.